plugins/jreleaser-gradle-plugin/src/main/groovy/org/jreleaser/gradle/plugin/internal/dsl/GitlabImpl.groovy

```diff
@@ -44,6 +45,8 @@ class GitlabImpl extends AbstractGitService implements Gitlab {
         changelog = objects.newInstance(ChangelogImpl, objects)
         milestone = objects.newInstance(MilestoneImpl, objects)
         commitAuthor = objects.newInstance(CommitAuthorImpl, objects)
+
+        identifier = objects.property(String).convention(Providers.notDefined())
```

Comment:
The `identifier` field is missing in `toModel()`

Reply:
My first change to non-gradle groovy code. I hope it fits now.